pages.es/common/golangci-lint.md

```diff
@@ -0,0 +1,16 @@
+# golangci-lint
+
+> Corredor de linters Go paralelizado, inteligente y rápido que se integra con los principales entornos de desarrollo integrado y soporta configuración en YAML.
+> Más información: <https://golangci-lint.run/usage/quick-start/>.
+
+- Ejecuta linters en la carpeta actual:
+
+`golangci-lint run`
+
+- Lista los linters habilitados y deshabilitados (Nota: los linters deshabilitados se muestran en último lugar, no los confundas con los habilitados):
```

Comment:
@tricantivu's suggestion seem to be marked as outdated, so created a new one for the same.
```suggestion
- Lista los linters habilitados y deshabilitados (Nota: los linters deshabilitados se muestran en el último lugar, no los confundas con los habilitados):
```